.github/workflows/raspbian.yml

```diff
@@ -27,6 +28,17 @@ jobs:
           workflow_id: deb.yml,docker.yml,raspbian.yml,rpm.yml,windows.yml
           access_token: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: 'Upgrade Ubuntu 20.04 -> 20.10 (to workaround "semop(1): encountered an error: Function not implemented")'
```

Comment:
This is only done to get a more recent version of `qemu-user-static`, is it?

As that package contains only statically linked binaries, it should be fine to just install that one package and skip the `dist-upgrade`. Another alternative would be to check if https://hub.docker.com/r/multiarch/qemu-user-static/ works.

Reply:
I wouldn't say that dist-upgrading a CI runner to an EOL release is obviously a better choice. Also, I mentioned an alternative that could work without any hacks.

Reply:
I could also upgrade -> HH -> II (II because HH will be EOL on JJ release), but just one upgrade seems just fine to me.